packages/tests/tests/FRAMEWORK+SERVER+drizzle.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { describeBati, exec, npmCli } from "@batijs/tests-utils";
+
+export const matrix = [["solid", "react", "vue"], ["express", "h3", "hono", "fastify"], "drizzle", "eslint"];
+
+await describeBati(({ test, expect, fetch, beforeAll }) => {
+  beforeAll(async () => {
+    await exec(npmCli, ["run", "drizzle:generate"]);
+    await exec(npmCli, ["run", "drizzle:migrate"]);
+    await exec(npmCli, ["run", "drizzle:seed"]);
+  }, 70000);
+
+  test("home", async () => {
+    const res = await fetch("/");
+    expect(res.status).toBe(200);
+    expect(await res.text()).not.toContain('{"is404":true}');
+  });
+
+  test("todo", async () => {
+    const res = await fetch("/todo");
+    expect(res.status).toBe(200);
+    expect(await res.text()).not.toContain('{"is404":true}');
+  });
+
+  test("/api/todo/create", async () => {
+    const res = await fetch("/api/todo/create", {
+      method: "POST",
+      body: JSON.stringify({ text: "test" }),
+      headers: {
+        "content-type": "application/json",
+      },
+    });
+    expect(res.status).toBe(201);
+    expect(await res.json()).toEqual({
+      message: "New Todo Created",
+      result: { changes: 1, lastInsertRowid: 3 },
+    });
+  });
+});
```

Comment:
Merge the 2 tests files into this one. This will limit the number of time spent by e2e tests to setup.